src/doc_table.c

```diff
@@ -303,34 +303,48 @@ void DocTable_RdbLoad(DocTable *t, RedisModuleIO *rdb, int encver) {
 
 void DocTable_AOFRewrite(DocTable *t, const char *indexName, RedisModuleIO *aof) {
   RedisModuleCtx *ctx = RedisModule_GetContextFromIO(aof);
+  Buffer tmpsBuf;
+  Buffer_Init(&tmpsBuf, 16);
+  char dblBuf[1024] = {0};
```

Comment:
1024 is waaaaay too much. DBL_DECIMAL_DIG specifies this correctly I think.

Reply:
Actually.. https://stackoverflow.com/questions/1701055/what-is-the-maximum-length-in-chars-needed-to-represent-any-double-value - guy claims it can get as big as 1079, some comments suggest it can get even larger!